reference_docs/YAML_models/calconnect_document_attributes.yaml

```diff
@@ -0,0 +1,64 @@
+version: v3.4.5 #Verify versions
+
+inherits_from:
+  # Also may be referred to as “parent flavor”
+  flavor: standoc
+  version: v1.x.x
+
+document_attributes:
+#--- Document attributes  ---#
+  docnumber: 
+    tags: ["Document attributes"]
+    title: "Document number" 
+    description: "CalConnect document number, as allocated by TCC.".
+
+  status: 
+    tags: ["Document attributes"]
+    title: "Status" 
+    description: |
+      The status of the document. Synonym: `:docstage:`.
+      Can be one of:
+
+      * proposal
+      * working-draft
+      * committee-draft
+      * draft-standard
+      * final-draft
+      * published
+      * withdrawn.
+    value_spec: "enum: proposal, working-draft, committee-draft, draft-standard, final-draft, published, withdrawn"
+
+  doctype: 
+    tags: ["Document attributes"]
+    title: "Document type" 
+    description: |
+      The type of the document. Can be one of:
+      +
+      --
+      standard::: Standard (no suffix on identifier; e.g. `CC 1`)
```

Comment:
This only works if you leave the description unchanged:

````
value_spec: "enum: standard, directive, guide, specification, report, administrative, amendment, technical corrigendum, advisory"
description: |
The type of the document. Can be one of:
+
--
standard:: Standard (no suffix on identifier; e.g. `CC 1`)
...
--
````

(We don't need the definition list embedded any more, by the way, with `:::`: its context is a YAML hash, not an Asciidoctor first-level definition list.)

It would be possible to embed definitions in the value_spec string. Let's not: that's making things unnecessarily convoluted.